client/src/components/auth/Signin.js

```diff
@@ -23,44 +29,38 @@ class Signin extends Component {
         <Helmet>
```

Comment:
I'm new to react still but what's Helmet?

Reply:
It's a library used to set a page's tab header, which is the name of the page you're on. It also allows different tab headers to be rendered based on the current state or props passed, if we want a page to have a dynamic tab header. In the image below, the tab headers are `YouTube` and `Google`.
![image](https://user-images.githubusercontent.com/28130187/68265322-b3dcdd00-0019-11ea-9912-0142ca8594c9.png)